secure_message/resources/health.py

```diff
@@ -43,7 +43,6 @@ def get():
         details = {'Name': current_app.config['NAME'],
                    'Version': current_app.config['VERSION'],
                    'SMS Log level': current_app.config['SMS_LOG_LEVEL'],
-                   'APP Log Level': current_app.config['APP_LOG_LEVEL'],
```

Comment:
in test_health_monitor.py I think it's looking for this, which is causing the test to fail.

Reply:
Thanks for fixing that